web/src/features/fireWatch/components/FireWatchDashboard.tsx

```diff
@@ -123,24 +199,37 @@ const FireWatchDashboard = () => {
           getRowClassName={params => `in-prescription-${params.row.inPrescription}`}
           initialState={{
             sorting: {
-              sortModel: [{ field: 'id', sort: 'asc' }]
+              sortModel: [
+                { field: 'status', sort: 'asc' },
+                { field: 'id', sort: 'asc' }
+              ]
             }
           }}
           sx={{
             '.in-prescription-all': {
-              bgcolor: FireWatchPrescriptionColors.all.bgcolor,
-              '&:hover': { bgcolor: FireWatchPrescriptionColors.all.hover }
+              bgcolor: `${FireWatchPrescriptionColors.all.bgcolor} !important`,
```

Comment:
Did you try using a more specific selector before going with `!important`? Maybe something like `.MuiDataGrid-row.in-prescription-all`.